src/index.js

```diff
@@ -72,11 +72,27 @@ function Logging(options) {
     return new Logging(options);
   }
 
+  // Determine what scopes are needed.
+  // It is the union of the scopes on all three clients.
+  let scopes = [];
```

Comment:
@lukesneeringer It doesn't seem the burden of this code should be on the client library. It looks like we're asking the generated library for information only to give it back.

Reply:
I probably agree but do not want to block on it.